core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -177,6 +177,12 @@ private TableProperties() {}
   public static final String AVRO_COMPRESSION_LEVEL_DEFAULT = null;
 
   public static final String ORC_STRIPE_SIZE_BYTES = "write.orc.stripe-size-bytes";
+
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "write.orc.bloom.filter.columns";
```

Comment:
Nit: Ideally there would be a default value, `ORC_BLOOM_FILTER_COLUMNS_DEFAULT`. Even if it’s null (like Avro compression level default) or empty string, I think it’s better for readability and safe refactoring later to have the default defined where the rest are.